compose.yaml

```diff
@@ -0,0 +1,41 @@
+services:
```

Comment:
License and Copyright header missing, please refer to [TRG 7.02 - License and Copyright header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02).